tests/actions/vstudio/cs2005/projectsettings.lua

```diff
@@ -13,17 +13,17 @@
 -- Setup
 --
 
-	local sln, prj
+	local wks, prj
 
 	function suite.setup()
 		_ACTION = "vs2005"
-		sln = test.createsolution()
+		wks = test.createWorkspace()
 		language "C#"
 		uuid "AE61726D-187C-E440-BD07-2556188A6565"
 	end
 
 	local function prepare()
-		prj = premake.solution.getproject(sln, 1)
+		prj = premake.solution.getproject(wks, 1)

```

Comment:
I'm getting there—because this is such a big change, I am trying to do it in stages. Because I am touching so many files, I want to push often so I don't end up with 100s of conflicts. And I also want to make sure that I'm not breaking things for other people as I go. 

Almost done though. One or two more pushes should do it for core, and then I will update the submodules.